services/http_gate/.http.env

```diff
@@ -0,0 +1,15 @@
+GW_LISTEN_ADDRESS=0.0.0.0:8080
```

Comment:
Protocol name prefix should be used here, IMO

Reply:
nope, it's not need here

Reply:
```
2020-11-27T17:47:24.363+0300    info    neofs-gw/app.go:219     run gateway server      {"address": "http://0.0.0.0:8080"}
2020-11-27T17:47:24.363+0300    fatal   neofs-gw/app.go:224     could not start server  {"error": "listen tcp4: address http://0.0.0.0:8080: too many colons in address"}
```

Reply:
Can't agree. All protocol gateways must have their own prefixes. For http protocol gateway it should be `HTTP_GW_` or something similar. Using GW is confusing, it may be interpreted as global option for all protocol gateways, which is not true.